### PR TITLE
Sm8450 sm8550 update for v6.4 & v6.5

### PR DESCRIPTION
**Pull request by superna9999**

### CODE REVIEW

File: _soc/sm8550.md [hunk 1]
```diff
@@ -35,12 +35,12 @@ status-cpu-ddrfreq:
 status-cpu-l3cache:
 status-cpu-llcc: 6.3
 status-cpu-smp:
-status-crypto-qcrypto: WIP
+status-crypto-qcrypto: 6.5
```
Comment (lumag) [inline _soc/sm8550.md]: next-6.5

File: _soc/sm8550.md [hunk 2]
```diff
@@ -35,12 +35,12 @@ status-cpu-ddrfreq:
...
 status-crypto-rng:
 status-debug-coresight:
 status-debug-dcc:
 status-debug-eud:
-status-display-dp: WIP
+status-display-dp: 6.5
```
Comment (lumag) [inline _soc/sm8550.md]: next-6.5

File: _pmic/pm8550.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ pmic-fuelgague: N/A
 pmic-gpio: 6.3
 pmic-haptics: N/A
 pmic-labib: N/A
-pmic-lpg: N
+pmic-lpg: 6.5
```
Comment (lumag) [inline _pmic/pm8550.md]: next-6.5

**Review by lumag — CHANGES_REQUESTED** [review]
Thanks! Could you please use 'next-6.n' so that is is clearly visible that it is not yet merged.

**Comment by superna9999** [comment]
Ack

**Review by lumag — APPROVED** [review]

**Pull request merged**